### DISCUSSION TITLE
post/podo-elk-threadcontext-part-3/

### DISCUSSION THREAD

**Discussion by giscus** [kakaopay/techlog-comments#102] [Announcements]
# post/podo-elk-threadcontext-part-3/

ELK 환경에서 Multi Thread Context를 사용하여, 이슈 트래킹 전략을 개선한 경험을 공유하는 글입니다.

https://tech.kakaopay.com/post/podo-elk-threadcontext-part-3/

**Comment by jws1837** [comment]
너무 좋네요.
